<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenderRecognition</class>
 <widget class="QWidget" name="GenderRecognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenderRecognition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 101, 164);</string>
  </property>
  <widget class="QLabel" name="lbl_show">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>741</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 233, 79);</string>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
